.github/workflows/build.yml

```
@@ -160,10 +160,10 @@ jobs:
 
       - name: Cache build result
         id: cache-build-result
-        uses: actions/cache@v2
+        uses: actions/cache@v3
         with:
           path: build/
-          key: ${{ matrix.artifact_name }}-v${{ env.ONNXRUNTIME_VERSION }}-cache-v1-${{ hashFiles('matrix.json') }}
+          key: ${{ matrix.artifact_name }}-v${{ env.ONNXRUNTIME_VERSION }}-cache-v3-${{ hashFiles('matrix.json') }}

```

ここの`v1`とかはキャッシュを破壊的に変更したくなった時に変えよう、みたいな感じで置いているのですが、まあそういうことが必要になったタイミングで足すのがいい気がしました。
意図不明になりがちなのでついでに削除しちゃう形がいいかなと･･･！
```suggestion
          key: ${{ matrix.artifact_name }}-v${{ env.ONNXRUNTIME_VERSION }}-cache-${{ hashFiles('matrix.json') }}

```

---

賛成です。リポジトリに対してwrite権限を持つなら個々のキャッシュは手動で消せるわけですし。